notifications/sms-quickstart/create-binding/example.curl

```diff
@@ -1,8 +1,6 @@
 curl -X POST 'https://notify.twilio.com/v1/Services/ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX/Bindings' \
---data-urlencode 'Endpoint=your_user@example.com:sms'  \
---data-urlencode 'Identity=your_user@example.com'  \
+--data-urlencode 'Endpoint=00000001:sms'  \ 
+--data-urlencode 'Identity=00000001'  \ # We recommend using a GUID or other anonymized identifier for Identity.
```

Comment:
Would omit the comment, as the primary use case for the curl snippets is copy/paste for testing.